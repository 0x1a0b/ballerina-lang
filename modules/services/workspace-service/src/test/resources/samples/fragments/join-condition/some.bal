some 1 Worker1, Worker2
