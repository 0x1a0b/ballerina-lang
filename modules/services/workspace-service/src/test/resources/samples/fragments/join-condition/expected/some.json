{"type":"join_statement","whitespace_descriptor":{"regions":{"0":"","1":"","2":"","3":" ","4":"","5":"","6":"","7":""},"children":{"joinWorkers":{"regions":{},"children":{"Worker1":{"regions":{"0":" ","1":""}},"Worker2":{"regions":{"0":" ","1":""}}}},"joinCondition":{"regions":{"0":"","1":" ","2":""}}}},"join_type":"some","join_count":1,"join_workers":["Worker1","Worker2"]}